### PR TITLE
Support java time Standard DateTime formatters

### PR DESCRIPTION
**Pull request by pchalcol**
## Summary
Use standard java `DateTimeFormatters` instances as valid timestamp types and handle timezones

**PR Type: Bug Fix**

**Status: Ready to review**

**Breaking change? Yes/No**
No

## Description
Standard iso datetime formatters, even if declared, weren't taken into account at ingestion time.
This PR fixes this issue and forces all time related data to be based on UTC timezone and not the system default one.

### How has this been tested?
- [x] unit tests
- [x] integration tests (data ingestion)

### Remaining Todos
None

## Contributor checklist:
Go over all the following points, and put an `x` in all the boxes that apply.
If you're unsure about any of these, don't hesitate to ask. We're here to help!
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/main/scala/com/ebiznext/comet/schema/model/PrimitiveType.scala [hunk 1]
```diff
@@ -247,6 +265,8 @@ object PrimitiveType {
         null
       else {
         val instant = instantFromString(str, timeFormat, zone)
+        println(s"instant = $instant")
```
Comment (hayssams) [inline src/main/scala/com/ebiznext/comet/schema/model/PrimitiveType.scala:268]: Wanna keep this println ?

File: src/main/scala/com/ebiznext/comet/schema/model/PrimitiveType.scala [hunk 2]
```diff
@@ -260,13 +280,15 @@ object PrimitiveType {
   import DateTimeFormatter._
 
   val dateFormatters: Map[String, DateTimeFormatter] = Map(
+    // ISO_LOCAL_TIME, ISO_OFFSET_TIME and ISO_TIME patterns are specific to time handling, without a date
```
Comment (hayssams) [inline src/main/scala/com/ebiznext/comet/schema/model/PrimitiveType.scala:283]: You're right let's remove them but keep the comment

File: src/test/scala/com/ebiznext/comet/schema/model/PrimitiveTypeSpec.scala [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+      println(s"Expected: ${expected}")
+      println(s"Actual: ${actual.asInstanceOf[Timestamp]}")
+
+      println(s"Expected long value: ${expected.getTime}")
+      println(s"Actual long value: ${actual.asInstanceOf[Timestamp].getTime}")
+
+      actual shouldEqual expected
+    }
+
+//    "Parsing a string with the ISO_LOCAL_DATE standard date pattern" should "return a valid timestamp" in {
```
Comment (hayssams) [inline src/test/scala/com/ebiznext/comet/schema/model/PrimitiveTypeSpec.scala:91]: dead code is dead

File: src/main/scala/com/ebiznext/comet/schema/model/PrimitiveType.scala (hunk 1 above)
Comment (pchalcol) [reply src/main/scala/com/ebiznext/comet/schema/model/PrimitiveType.scala:268]: Nope I don't intend to. Just need to finish and fix all unit tests, remove all remaining printlns before removing the WIP status.

**Pull request merged**